### PR TITLE
MOB-1765 Created service to track selected wallet

### PR DESCRIPTION
**Pull request by Oleg-Pecheneg**

### CODE REVIEW

File: unstoppable-ios-app/domains-manager-ios/Entities/NFTDisplayInfo.swift [hunk 1]
```diff
@@ -75,7 +73,7 @@ struct NFTDisplayInfo: Hashable, Identifiable {
         
 //                    try? await Task.sleep(seconds: TimeInterval(arc4random_uniform(5)))
```
Comment (rommex) [inline unstoppable-ios-app/domains-manager-ios/Entities/NFTDisplayInfo.swift:74]: dead code?

File: unstoppable-ios-app/domains-manager-ios/Services/CoreAppCoordinator/CoreAppCoordinator.swift [hunk 2]
```diff
@@ -40,8 +41,9 @@ extension CoreAppCoordinator: CoreAppCoordinatorProtocol {
         setOnboardingAsRoot(flow)
     }
     
-    func showHome(mintingState: DomainsCollectionMintingState) {
-        setDomainsCollectionScreenAsRoot(mintingState: mintingState)
+    func showHome(mintingState: DomainsCollectionMintingState, wallet: WalletEntity) {
+//        setDomainsCollectionScreenAsRoot(mintingState: mintingState)
```
Comment (rommex) [inline unstoppable-ios-app/domains-manager-ios/Services/CoreAppCoordinator/CoreAppCoordinator.swift:45]: dead?

**Review by rommex — APPROVED** [review]
with minor comments

File: unstoppable-ios-app/domains-manager-ios/Services/CoreAppCoordinator/CoreAppCoordinator.swift (hunk 2 above)
Comment (Oleg-Pecheneg) [reply unstoppable-ios-app/domains-manager-ios/Services/CoreAppCoordinator/CoreAppCoordinator.swift:45]: Temporarily

**Pull request merged**